src/dom-updates.js

```diff
@@ -0,0 +1,150 @@
+let domUpdates = {
```

Comment:
This is awesome how domUpdates mimics a class with methods, but is just an object!  Awesome!